klippy/extras/bme280.py

```diff
@@ -8,6 +8,8 @@
 
 REPORT_TIME = .8
 BME280_CHIP_ADDR = 0x76
+BMP180_CHIP_ADDR = 0x77
```

Comment:
This constant doesn't appear to be used anywhere in the module.

Reply:
Indeed, it was mostly a note for documenting. Removed.